packages/service-clients/odsp-client/src/odspClient.ts

```diff
@@ -83,7 +83,7 @@ async function getWebsocketToken(
  * These values will only be used if the feature gate is not already set by the supplied config provider.
  */
 const odspClientFeatureGates = {
-	// None yet
+	"Fluid.Driver.Odsp.enableLargeBlobUpload": false,
```

Comment:
Can we have that only in Designer, not for all ODSP clients? It does not belong here, in a sense that if validating with Designer goes Ok, the right next thing is to enable it across whole repo (and all clients), not just OdpsClients. I.e. Loop (as an example) should benefit from it too (at some point)

Reply:
I removed it. Will add it to designer repo.